src/redux/reducers/eventHandler.js

```diff
@@ -78,7 +62,22 @@ export default (state = initialState, action) => {
       };
     }
     case RESET_INPUT_FIELDS: {
-      return initialState;
+      return {
```

Comment:
This is a a duplication please find a way to solve it.

Reply:
@kagabof , this is resolved!